LibTrixi.jl/src/LibTrixi.jl

```diff
@@ -1,5 +1,8 @@
 module LibTrixi
 
+using OrdinaryDiffEq
+using Trixi
```

Comment:
We usually frown upon just blindly `using SomePackage` since that pulls in everything from that namespace that has been `export`ed from said package, and thus it is not possible/easy to decide anymore from which package a particular function is (at least not by just looking at the source code).
Thus, I propose to change this part of the code as follows:
```suggestion
using OrdinaryDiffEq: OrdinaryDiffEq, step!, check_error
using Trixi: Trixi
```